lib/nodeJavaBridge.js

```diff
@@ -21,6 +21,7 @@ java.nativeBindingLocation = binaryPath;
 
 var syncSuffix = undefined;
 var asyncSuffix = undefined;
+var readOnlyFunctionProperties = ['name'];
```

Comment:
Instead of this variable here, should the import function take this as an options?